Libraries/Components/WebView/WebView.android.js

```diff
@@ -137,6 +137,12 @@ var WebView = React.createClass({
      * Used to locate this view in end-to-end tests.
      */
     testID: PropTypes.string,
+
+    /**
+     * Determines whether HTML5 audio & videos require the user to tap before they can
+     * start playing. The default value is `false`
```

Comment:
nit: "The default value is `false`." is a sentence. Please add a period at the end. Sorry if I've missed that in the previous review.